### PR TITLE
BONNIE-1154 Fixes whitespace issue.

### PR DESCRIPTION
**Pull request by alexanderelliott121**
Pull request changes the way the elm is parsed to make it resemble the human readable.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1154
- [X] JIRA ticket links to this PR
- [X] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [X] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [X] Tests are included and test edge cases
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [X] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA test links: https://jira.mitre.org/browse/BONNIE-1177
- [X] Justification for using JIRA tests: Pure gui change, difficult to unit test.
- [X] JIRA tests have been added to sprint

**Reviewer 1:**

Name: @c-monkey
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: lib/measures/elm_parser.rb [hunk 1]
```diff
@@ -45,21 +45,14 @@ def self.parse_node(node)
...
-            if (/^define/ =~ child.to_html)
-              define_name = child.to_html.split("\"")[1]
-            end
-            clause = {
-              text: child.to_html
-            }
-            add_whitespace(clause)
-            clause[:ref_id] = child['r'] unless child['r'].nil?
-            ret[:children] << clause
+          if (/^define/ =~ child.to_html)
```
Comment (c-monkey) [inline lib/measures/elm_parser.rb:48]: the ruby style guide suggests parenthesis around conditionals is inappropriate and it doesn't match the style of the rest of this file

**Review by c-monkey — APPROVED** [review]
minor feedback, I don't care if we fix it, it's just a style suggestion, nothing functional

2 spaces matches human readable well

**Review by lukoktonos — CHANGES_REQUESTED** [review]
The JIRA test's image currently matches the `cql4bonnie` behavior for me:
image

This is what it looks like for me on this branch:
image

I believe the latter is correct and the JIRA test just needs to be updated.

**Comment by edeyoung** [comment]
This is what i see in the jira test:

image


It looks like your latter image. Maybe the test got updated since you last checked? Can you take another look, @losborne?

**Comment by lukoktonos** [comment]
@edeyoung When I am on the branch `BONNIE-1154_WhitespaceFix`, it looks like this:
image

The JIRA test says it should look like this when on that branch:
image

There are 2 or 4 extra spaces on the 2nd line.

**Review by lukoktonos — APPROVED** [review]
Looks good.  JIRA Test screenshot updated.
